### ISSUE TITLE
[BUG] After upgrade setting the source produces error "type 'double' is not a subtype of type 'int' in type cast "

### ISSUE THREAD

**Issue by jbworldllc** [jhomlala/betterplayer#188]
**Describe the bug**
After upgrading to  ^0.0.41 ..When setting the DataSource, I get error "type 'double' is not a subtype of type 'int' in type cast"

**To Reproduce**
 betterPlayerController = BetterPlayerController(betterPlayerConfiguration);

 BetterPlayerDataSource dataSource = BetterPlayerDataSource(
 BetterPlayerDataSourceType.network, "MEDIA URL HERE");

 betterPlayerController.setupDataSource(dataSource);

**Expected behavior**
Set datasource and play media.

**Screenshots**
https://ibb.co/jrXVQdB

**Flutter doctor**
type 'double' is not a subtype of type 'int' in type cast
*Screenshot*
https://ibb.co/12f3pcH

**Better Player version**
- Version: ^0.0.41

**Comment by jbworldllc** [comment]
Got it to work after editing code on 
method_channel_media_dart.dart
line : 202
TO : size: Size(map['width'] ?? 0.0, map['height'] ?? 0.0),

**Comment by samo92** [comment]
Hi, where can I find it?

**Comment by kkdlau** [comment]
> Got it to work after editing code on
> method_channel_video_player.dart
> line : 202
> TO : size: Size(map['width'] ?? 0.0, map['height'] ?? 0.0),

Thx, this perfectly solved the bug!

@samo92 the cached plugin is stored inside the `flutter/.pub-cache/hosted/pub.dartlang.org/`, you can locate the plugin and modify the source code.

**Comment by kkdlau** [comment]
Just one more thing, did anyone try using `BetterPlayerDataSource.file()` to configure controller's playlist?

I looked at the implemented of that method, it seem set the `BetterPlayerDataSourceType` to `BetterPlayerDataSourceType.network`, instead of `BetterPlayerDataSourceType.file`. And if I use constructor and explicitly set it like below:

```dart
BetterPlayerDataSource(BetterPlayerDataSourceType.file, localDocumentPath);
```

it works perfectly. Does anyone have ideas on that?

**Comment by jhomlala** [comment]
@jbworldllc @kkdlau fixed issues in 0.0.42.
